apps/discussions/app/App.js

```diff
@@ -1,41 +1,18 @@
 import React from 'react'
-import styled from 'styled-components'
-import { EmptyStateCard, GU, Header, Main, breakpoint } from '@aragon/ui'
-import nothingToSeeHere from './empty-state.png'
-
-const illustration = <img src={nothingToSeeHere} alt="" height="160" />
 
 function App() {
   return (
-    <Main>
```

Comment:
If we get rid of the Main component, the app will not look like as an aragon app (css, fonts, etc)